### DISCUSSION TITLE
Can the Whisper Model generate caption files?

### DISCUSSION THREAD

**Discussion by ryanpauldev** [deepgram/community#231] [General help] [answered]
I am working on a project that requires the creation of caption files. I am using one of Deepgram's older models to generate these captions right now, and I want to upgrade to using the new Whisper model. I've seen the Whisper Model generate transcripts, but not captions. Can anyone here help me?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @ryanpauldev, our Node and Python SDKs have built-in functionality for generating SRT and WebVTT caption files. These methods work with any of our models, including Whisper. 

Note that we recommend using `'utterances': True` for best results, as this allows captions to conform to pauses in speech.

Node documentation example: [Automatically Generating WebVTT and SRT Captions](https://developers.deepgram.com/docs/automatically-generating-webvtt-and-srt-captions)
See the recent Python PR for more Python examples: https://github.com/deepgram/deepgram-python-sdk/pull/95

Proof of concept in Python, specifically with Whisper:
```
from deepgram import Deepgram
deepgram = Deepgram("<API Key>")
response = deepgram.transcription.sync_prerecorded({"url": "https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav"}, {'model': 'whisper', 'utterances': True})
deepgram.extra.to_SRT(response)

1
00:00:00,060 --> 00:00:00,560
yep

2
00:00:07,019 --> 00:00:08,699
i said it before and i'll say it

3
00:00:08,699 --> 00:00:09,199
again

4
00:00:09,960 --> 00:00:11,439
life moves pretty fast

5
00:00:12,039 --> 00:00:13,719
you don't stop and look around once in

6
00:00:13,719 --> 00:00:15,759
a while you could miss it
```
